Applications/Tizen_native/CustomShortcut/res/edje/main.edc

```diff
@@ -100,20 +100,40 @@ collections {
     }
     programs {
       PROGRAM_BUTTON("home/to_train", "routes/to", "select")
-      PROGRAM_BUTTON("home/to_test", "routes/to", "draw")
+      PROGRAM_BUTTON("home/to_test", "routes/to", "draw:inference")
     }
   }
   group {
     name: "select";
     parts {
       PART_TITLE("select/title", "Select target emoji...")
+      PART_BUTTON("select/sad", "😢", 0.1, 0.3, 0.45, 0.7)
```

Comment:
I wonder whether it is ok to have "😢". Is this because of github editor?

Reply:
Tested and it works fine on the wearable app